proto/swap_resolver.proto

```diff
@@ -0,0 +1,106 @@
+
+syntax = "proto3";
+
+//import "google/api/annotations.proto";
+
+//option java_multiple_files = true;
+//option java_package = "io.grpc.examples.routeguide";
+//option java_outer_classname = "RouteGuideProto";
+
+package swapresolver;
+
+// Interface exported by the server.
+service SwapResolver {
+  // ResolveHash is used by LND to request translation of Rhash to a pre-image.
+  // the resolver may return the preimage and error indicating that there is no
+  // such hash/deal
+  rpc ResolveHash(ResolveReq) returns (ResolveResp) {}
+
+}
+
+enum CoinType {
+    BTC = 0;
+    LTC = 1;
+}
+
+// 6
+message ResolveReq {
+  string hash = 1 ;
+}
+
+message ResolveResp {
```

Comment:
Better not. This is going to be part of LND and It better be consistent with them.

Reply:
This will be part of LND? Or only used by it?

And from what I can see, the lnd proto definition uses fully spelled out "Request" and Response" as well, I've been trying to follow that naming convention. Are you looking at something different?